components/perl/Date-Calc/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+#
+
+BUILD_STYLE = makemaker
+BUILD_BITS = 32_and_64
+USE_COMMON_TEST_MASTER = yes
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=			Date-Calc
+COMPONENT_VERSION=		6.4
+COMPONENT_FMRI=			library/perl-5/date-calc
+COMPONENT_SUMMARY=		Date::Calc - Gregorian calendar date calculations
+COMPONENT_CLASSIFICATION=	Development/Perl
+COMPONENT_SRC=			$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=		$(COMPONENT_SRC).tar.gz
+COMPONENT_PROJECT_URL=		https://metacpan.org/dist/TimeDate
+COMPONENT_ARCHIVE_HASH=		sha256:7ce137b2e797b7c0901f3adf1a05a19343356cd1f04676aa1c56a9f624f859ad
+COMPONENT_ARCHIVE_URL=		https://cpan.metacpan.org/authors/id/S/ST/STBEY/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=		Artistic
+COMPONENT_LICENSE_FILE=		license/Artistic.txt
+
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_TEST_TRANSFORMS += \
+        '-e "/^\#/d"' \
+        '-e "/^PERL/d"' \
+        '-e "/^make/d"'
+
+REQUIRED_PACKAGES += library/perl-5/carp-clan
+REQUIRED_PACKAGES += library/perl-5/bit-vector
+
```

Comment:
PERL_REQUIRED_PACKAGES needed here?

Reply:
I tried initially but pkg5 did not get updated accordingly. So decided for REQUIRED_PACKAGES. But maybe  I did not understand correctly the PERL_ variant.

Reply:
> I tried initially but pkg5 did not get updated accordingly. So decided for REQUIRED_PACKAGES. But maybe I did not understand correctly the PERL_ variant.

It should work.  Let me check...

Reply:
I changed the `Makefile` and ran `gmake publish`.  Below is the result (as expected):

```
diff --git a/components/perl/Date-Calc/Makefile b/components/perl/Date-Calc/Makefile
index adf9edd2f..4303a4e91 100644
--- a/components/perl/Date-Calc/Makefile
+++ b/components/perl/Date-Calc/Makefile
@@ -39,8 +39,8 @@ COMPONENT_TEST_TRANSFORMS += \
         '-e "/^PERL/d"' \
         '-e "/^make/d"'
 
-REQUIRED_PACKAGES += library/perl-5/carp-clan
-REQUIRED_PACKAGES += library/perl-5/bit-vector
+PERL_REQUIRED_PACKAGES += library/perl-5/carp-clan
+PERL_REQUIRED_PACKAGES += library/perl-5/bit-vector
 
 # Auto-generated dependencies
 REQUIRED_PACKAGES += runtime/perl-522
diff --git a/components/perl/Date-Calc/pkg5 b/components/perl/Date-Calc/pkg5
index c3dddd244..077d823c0 100644
--- a/components/perl/Date-Calc/pkg5
+++ b/components/perl/Date-Calc/pkg5
@@ -2,7 +2,13 @@
     "dependencies": [
         "SUNWcs",
         "library/perl-5/bit-vector",
+        "library/perl-5/bit-vector-522",
+        "library/perl-5/bit-vector-524",
+        "library/perl-5/bit-vector-534",
         "library/perl-5/carp-clan",
+        "library/perl-5/carp-clan-522",
+        "library/perl-5/carp-clan-524",
+        "library/perl-5/carp-clan-534",
         "runtime/perl-522",
         "runtime/perl-524",
         "runtime/perl-534",
```

Reply:
BTW, I started with exactly your dataset:

```
commit 742a78fe06f8a6bb3a5d24b60cd23e9d4a1c5316 (HEAD -> TEST)
Author: fritzkink <friedrich.kink@fkink.de>
Date:   Fri Jun 10 17:13:08 2022 +0200

    Add Date::Calc perl module
```

Reply:
ok tried once more and it seems that a reboot of my zone solved my issue. Works as you described. Thanks for the support.